tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -32,54 +32,40 @@
 
 
 @pytest.mark.parametrize(
-    "callbacks,refresh_rate",
+    "kwargs",
     [
-        ([], None),
-        ([], 1),
-        ([], 2),
-        ([TQDMProgressBar(refresh_rate=1)], 0),
-        ([TQDMProgressBar(refresh_rate=2)], 0),
-        ([TQDMProgressBar(refresh_rate=2)], 1),
+        # won't print but is still set
+        {"callbacks": TQDMProgressBar(refresh_rate=0)},
+        {"callbacks": TQDMProgressBar()},
+        # callbacks takes precedence
```

Comment:
I'm assuming that passing the instance as a callback should take precedence over `enable_progress_bar`